test/test_contract.py

```diff
@@ -1,3 +1,7 @@
+import os, sys
```

Comment:
I use notepad++ and cmd, no magic, and I get import error without these lines if I run pytest. What do you think. Also, this is the first test file, so its not necessary right now, but would you add these path lines to the beginning of the rest of the tests?